apps/chat/src/pages/_app.tsx

```diff
@@ -14,10 +14,10 @@ import { HomeProps } from '.';
 import { createStore } from '@/src/store';
 import '@/src/styles/globals.css';
 
-const inter = Inter({
+export const inter = Inter({
```

Comment:
do not export this inter font - create separate const for it and use it where it needed
Import something from FE to server code is bad practise

Reply:
Otherwise it doesn't work.